tasks/setup_infrared.yml

```diff
@@ -9,7 +9,10 @@
       - libffi-devel
       - python-virtualenv
       - libselinux-python
+      - python30-virtualenv
+      - python3-libselinux
   become: true
+  failed_when: false
```

Comment:
Why is it OK to skip package installs?

Reply:
For now this will be fine. We will enhance it later